src/qt/guiutil.cpp

```diff
@@ -165,6 +167,13 @@ bool parseBitcoinURI(const QUrl &uri, SendCoinsRecipient *out)
             rv.label = i->second;
             fShouldReturnFalse = false;
         }
+        if (i->first == "IS")
+        {
+            if(i->second.compare(QString("1")) == 0)
+                rv.fUseInstantSend = true;
```

Comment:
Most probably yes. 
But in an OO context I prefer to not rely on overloaded operators when an object offers an explicit  comparator.
But I can change it if you want, no problem.